uwuno/src/test/java/com/learning/uwuno/integration/tests/basicPlayerTest.java

```diff
@@ -0,0 +1,113 @@
+//package com.learning.uwuno.integration.tests;
```

Comment:
Commit for accident?

Reply:
i'll add it later. lol just ignore for now